vmware_aria_operations_integration_sdk/util.py

```diff
@@ -20,25 +21,53 @@ def __get__(self, instance: Optional[object], owner: Optional[type[object]]) ->
         return instance.__dict__[self.attribute_name]
 
 
-def port_range(port: str) -> int:
-    try:
-        val = int(port)
-    except ValueError:
-        raise argparse.ArgumentTypeError(f"Invalid integer value: {port}")
+class RangeAction(argparse.Action):
+    def __init__(
+        self,
+        option_strings: list[str],
+        dest: str,
+        nargs: Optional[int] = None,
+        const: Any = None,
+        default: Any = None,
+        type: Any = None,
+        choices: Any = None,
+        required: bool = False,
+        help: Optional[str] = None,
+        metavar: Union[str, None] = None,
+        lower_bound: int = 0,
+        upper_bound: int = 100,
```

Comment:
* For the arguments we don't specifically use, can we just use `**kwargs`?
* Does it work to set `metavar` here programmatically based on lower_bound and `upper_bound`?
* If so, and we keep `metavar` in the argument list, can it's type be `Optional[str]`?